test/functional/net.py

```diff
@@ -13,9 +13,7 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.authproxy import JSONRPCException
 from test_framework.util import (
-    assert_equal,
-    start_nodes,
-    connect_nodes_bi,
+    assert_equal, connect_nodes_bi, p2p_port, start_nodes,
```

Comment:
git solves conflicts on a line basis, not word basis, so I'd prefer to keep those in separate lines to make it easier to solve merge conflicts (in case they pop up).